README.md

```diff
@@ -143,15 +143,9 @@ Key documents
 * Link to the Worker Portal User Acceptance Testing Environment: https://uat.myaries.alaska.gov 
 
 ## Technology requirements
-We are working in the Microsoft universe. We prefer:
+We are working with the Microsoft technology stack. Please see [DevSecOps.md](/DevSecOps.md) for specific requirements.
```

Comment:
There is another PR open which renames that document to DevSecOps: https://github.com/AlaskaDHSS/acq-alaska-dhss-modernization/pull/227

Once that PR merges then this link will be correct.

Reply:
Cool - thanks @ctro!

Reply:
OTOH, if it makes more sense to ditch the DevSecOps document and just paste that contents under "Technology requirements" let me know..

Reply:
Nah, I think the DevSecOps doc makes sense.